lib/tether/executer.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
Should be in `cmd/tether` - use the systemd-haveged in the appliance as is currently the case.